Governing Board Public Minutes/2023-10-23.md

```diff
@@ -0,0 +1,688 @@
+* Health Check Update
+    * KW presented an update on the repo checklist.
+* Scorecard
+    * AM presented an update on OpenSSF Scorecard.
+    * Volunteer for front end development & product management, plus demo of remediation tooling.
+    * Idea for trusted OpenSSF Scorecard weighting templates for orgs to select from rather than customize from scratch.
+    * **Action: Security scoring across multiple committees; add as a discussion item to the next governance meeting agenda.**
+* SOSS Update
+    * HT presented an update of the panels and activities from the SOSS event.
+    * Looking into the possibility of a SOSS EU summit, any assistance that can be provided is welcome.
+    * Task forces had a broad scope with approaching each subject. 
+    * Task forces established in this way to anchor on the things that the membership continues to support and has the capability to deliver on during the next year. Can discontinue work that stops making sense to do.
+    * Plan to continue SOSS-US and SOSS-EU annually.
+* Budget Update
+    * HT presented the proposed 2024 budget.
+    * Comment to be very deliberate with attendance at SecurityCon and similar OS security events.
+    * Looking to bring together projects across the LF for SecurityCon (name may change) currently we have a hold for the third weekend of October in 2024.
+    * Comment about OpenSSF engaging with more communities to position as a central OS security source.
+    * VulnCon mentioned - What does LF Staff participation look like? Is this attending/participating or is it sponsoring these types of events? If it is sponsoring, then tradeoff discussions will be required.
+    * Event strategy should identify the target audience for the OpenSSF and work to create our own must-attend event, bringing more foundations together in the lens of security focus.
+    * Question about revenue which is expected to be flat. Current budget exceeds revenue expectations which will have to be adjusted long term.
+
+### Resolving an Identity Crisis: What’s Next for OpenSSF, 2024 Planning and Prioritization
+
+OA presented newly GB approved Mission, Vision, and Strategy,  quotes, and different context types to keep in mind during discussion on future direction of OpenSSF. Types of work currently supported by OpenSSF were outlined in terms of cost, time, and impact. Concern that funding to support security initiatives will decrease as trends change and want to make sure OpenSSF is supporting initiatives that provide higher return and sustainability. The listed criteria are important, but including aspects of sustainability and increased commitment are also important to consider. It was noted that there is dependency between the types of work and want to clarify the duration [of types of work] to better inform the scope. Convening is important to build trust needed for the other types of work, particularly internet-facing services.  \
```

Comment:
```suggestion
OA presented newly GB approved Mission, Vision, and Strategy, quotes, and different context types to keep in mind during discussion on future direction of OpenSSF. Types of work currently supported by OpenSSF were outlined in terms of cost, time, and impact. Concern that funding to support security initiatives will decrease as trends change and want to make sure OpenSSF is supporting initiatives that provide higher return and sustainability. The listed criteria are important, but including aspects of sustainability and increased commitment are also important to consider. It was noted that there is dependency between the types of work and want to clarify the duration [of types of work] to better inform the scope. Convening is important to build trust needed for the other types of work, particularly internet-facing services.
```

Reply:
This is from the automated docs to md conversion we use.